src/feature/cache-table/cache-panel-head.tsx

```diff
@@ -100,19 +101,31 @@ export const CachePanelHead = ({
     }
   };
 
+  const isActiveSortProperty = sorting?.key === sortingProperty;
+  const ariaSortAttribute: AriaSortAttribute = isActiveSortProperty
```

Comment:
Forcing this type is pointless, let TS infer it.
```suggestion
  const ariaSortAttribute = isActiveSortProperty
```